packages/core/src/components/Combobox/__stories__/Combobox.mdx

```diff
@@ -192,6 +192,12 @@ We are using Combobox component for our board person picker.
 
 <Canvas of={ComboboxStories.ComboboxAsPersonPicker} />
 
+### Combobox on dark theme
+
+On dark theme, our combobox have a stroke instead of the drop shadow. This design created to enhance better visibility and differentiation between the background layer to the combobox itself.
```

Comment:
This can be misleading if it's experimental, no?

Reply:
I agree that we should wait with posting those stories, that's what design wants for other designers to see that